src/realm/object_id.hpp

```diff
@@ -31,16 +31,22 @@ class ObjectId {
      * Constructs an ObjectId with all bytes 0x00.
      */
     ObjectId() noexcept;
-    ObjectId(null) noexcept : ObjectId()
```

Comment:
Thanks. I had been considering deleting this, now that I understand more how `null` is used and that this isn't actually a null constructor (since there isn't one).